### PR TITLE
[Fix] update save_vtk_streamlines and load_vtk_streamlines

### PR DESCRIPTION
**Pull request by skoudoro**
This PR should fix the current issue with VTK9.  I just replace the old code with [FURY](https://github.com/fury-gl/fury) functions. This will make maintenance easier.

this PR supersede #2175

### CODE REVIEW

**Comment by arokem** [comment]
This makes sense to me. If you're reading VTK files, you probably don't mind depending on VTK and FURY. 

Does this require a recent version of FURY?

**Comment by skoudoro** [comment]
FURY >= 0.6.0 (last release)  will manage well VTK  >= 8.1.2

FURY < 0.6.0 will not work with VTK >= 9.0.0

**Comment by skoudoro** [comment]
But those functions exist on all FURY version

**Comment by arokem** [comment]
Cool! Do we want to check versions and provide a warning of some kind (e.g., if you have VTK 9 and FURY < 0.6, we raise a warning recommending to upgrade FURY). This might be overkill.

**Comment by skoudoro** [comment]
>  Do we want to check versions and provide a warning of some kind (e.g., if you have VTK 9 and FURY < 0.6, we raise a warning recommending to upgrade FURY). This might be overkill

warning added [here](https://github.com/dipy/dipy/pull/2196/files#diff-6b0a4657b986e812251c3e6b39d63eb7R20-R28)

**Comment by arokem** [comment]
OK - I am +1 for the merge. Anyone else need to take a look?

File: dipy/viz/__init__.py [hunk 1]
```diff
@@ -16,6 +17,16 @@
     from fury.data import (fetch_viz_icons, read_viz_icons,
                            DATA_DIR as FURY_DATA_DIR)
 
+    HAVE_VTK_9_PLUS = vtk.vtkVersion.GetVTKMajorVersion() >= 9
+    HAVE_FURY_0_6_LESS = LooseVersion(fury.__version__) < LooseVersion('0.6.0')
+    if HAVE_VTK_9_PLUS and HAVE_FURY_0_6_LESS:
+        msg = "You are using VTK 9+ and FURY {}. ".format(fury.__version__)
+        msg += "Some functions might not work. To have a better experience, "
+        msg += "we recommend you to upgrade FURY version (0.6.0+). "
```
Comment (Garyfallidis) [inline dipy/viz/__init__.py:25]: please upgrade to FURY version 0.6.0+.

**Pull request merged**